projects/ion/src/lib/icon/svgs/icons.ts

```diff
@@ -89,7 +94,7 @@ export const iconsPaths: Record<string, string> = {
   left3,
   'message-ballon': messageBallon,
   pencil,
-  'question-rounded': questionRounded,
```

Comment:
Take care with this change, it could affect other projects that are already using them with the old name.

Reply:
Very well placed. Icons already present will remain with the old naming